doorstop/run_bepinex.sh

```diff
@@ -5,73 +5,87 @@
 #
 # Usage: Configure the script below and simply run this script when you want to run your game modded.
 
-if [ -z "$1" ]; then echo "Please open run.sh in a text editor and configure executable name. Comment or remove this line when you're done." && exit 1; fi
-
 # -------- SETTINGS --------
 # ---- EDIT AS NEEDED ------
 
 # EDIT THIS: The name of the executable to run
 # LINUX: This is the name of the Unity game executable 
 # MACOS: This is the name of the game app WITHOUT the .app
```

Comment:
I think we can change that on macOS you **must** specify `.app` prefix for executable names, see comment below.